src/linux/BTSerialPortBindingServer.cc

```diff
@@ -385,65 +401,70 @@ void BTSerialPortBindingServer::Advertise(listen_baton_t * baton) {
 
     uint8_t rfcomm_channel = (uint8_t) baton->listeningChannelID;
 
-    const char *service_name = "RFCOMM Server socket";
-    const char *service_dsc = "A RFCOMM listening socket";
-    const char *service_prov = "Service Provider";
+    const char *service_name = "Serial Port";
+    const char *service_dsc = "An RFCOMM listening socket";
 
-    uuid_t root_uuid, l2cap_uuid, rfcomm_uuid;
-    sdp_list_t *l2cap_list = 0, 
+    uuid_t root_uuid, service_uuid, l2cap_uuid, rfcomm_uuid;
+    sdp_list_t *l2cap_list = 0,
                *rfcomm_list = 0,
                *root_list = 0,
-               *proto_list = 0, 
+               *proto_list = 0,
+               *service_class_list = 0,
                *access_proto_list = 0;
     sdp_data_t *channel = 0;
 
     sdp_record_t *record = sdp_record_alloc();
 
     sdp_set_service_id(record, baton->uuid);
 
+    sdp_uuid16_create(&service_uuid, _SPP_UUID);
```

Comment:
Couldn't found any documentation about what the registers of the SDP actually means, so after looking at the output of some other services, feels to me like Service Class ID and Service ID should point the same service data in case we are exposing a standard service like Serial Port, so I'm gonna change it to show "Serial Service" for the service name in case we are exposing the standard service, or "RFCOMM custom service" and the  UUID in the Service Class ID register.
Output example of our custom service:
```
Service Name: RFCOMM custom service
Service Description: An RFCOMM listening socket
Service RecHandle: 0x10008
Service Class ID List:
  UUID 128: 38e851bc-7144-44b4-9cd8-80549c6f2912
Protocol Descriptor List:
  "L2CAP" (0x0100)
  "RFCOMM" (0x0003)
    Channel: 10
```
Serial Port service output:
```
Service Name: Serial Port
Service Description: An RFCOMM listening socket
Service RecHandle: 0x10008
Service Class ID List:
  "Serial Port" (0x1101)
Protocol Descriptor List:
  "L2CAP" (0x0100)
  "RFCOMM" (0x0003)
    Channel: 10
```